### PR TITLE
More description in the content

### PR DESCRIPTION
**Pull request by joinimran**
Added relevant information in the content to make things more clear.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5978

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: devices/hololens/hololens-provisioning.md [hunk 1]
```diff
@@ -53,8 +53,8 @@ Provisioning packages can include management instructions and policies, customiz
 
 ### 1. Install Windows Configuration Designer on your PC. (There are two ways to do this).
 
-1. **Option 1:** [From Microsoft Store](https://www.microsoft.com/store/apps/9nblggh4tx22)
-2. **Option 2:** [From the Windows Assessment and Deployment Kit (ADK) for Windows 10](https://developer.microsoft.com/windows/hardware/windows-assessment-deployment-kit). If you install Windows Configurations Designer from the Windows ADK, select **Configuration Designer** from the **Select the features you want to install** dialog box.
+1. **Option 1:** [From Microsoft Store](https://www.microsoft.com/store/apps/9nblggh4tx22). This includes HoloLens 2 capabilities.
+2. **Option 2:** [From the Windows Assessment and Deployment Kit (ADK) for Windows 10](https://developer.microsoft.com/windows/hardware/windows-assessment-deployment-kit). If you install Windows Configurations Designer from the Windows ADK, select **Configuration Designer** from the **Select the features you want to install** dialog box. This option does not include HoloLens 2 capabilities.
```
Comment (JohanFreelancer9) [inline devices/hololens/hololens-provisioning.md]:
```suggestion
2. **Option 2:** [From the Windows Assessment and Deployment Kit (ADK) for Windows 10](https://developer.microsoft.com/windows/hardware/windows-assessment-deployment-kit). If you install Windows Configuration Designer from the Windows ADK, select **Configuration Designer** from the **Select the features you want to install** dialog box. This option does not include HoloLens 2 capabilities.
```

**Comment by joinimran** [comment]
@mypil - Ready to go live

**Comment by illfated** [comment]
#sign-off

**Pull request merged**